client/state/stats/lists/utils.js

```diff
@@ -14,3 +16,24 @@ import toPairs from 'lodash/toPairs';
 export function getSerializedStatsQuery( query = {} ) {
 	return JSON.stringify( sortBy( toPairs( query ), ( pair ) => pair[ 0 ] ) );
 }
+
+export const Parsers = {
```

Comment:
Based on our variable naming guidelines:

> Constructors intended for use with new should have a capital first letter (UpperCamelCase).

> Variables intended to be used as a constant can be defined with the SCREAMING_SNAKE_CASE naming convention. Note that while any variable declared using const could be considered a constant, in the context of our application this usage should usually be limited to top-level or exported module values.

Since this isn't a constructor intended for use with `new`, I think this should be either `PARSERS` (constant) or `parsers` (regular variable export).  Another option would be a separate file.